longevity_mv_synchronous_updates_test.py

```diff
@@ -0,0 +1,63 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2020 ScyllaDB
+
+# This is stress longevity test that runs light weight transactions in parallel with different node operations:
+# disruptive and not disruptive
+#
+# After the test is finished will be performed the data validation.
+
+from longevity_test import LongevityTest
+from sdcm.sct_events.database import DatabaseLogEvent
+from sdcm.sct_events.filters import DbEventsFilter
+from sdcm.utils.common import keyspace_min_max_tokens
+from sdcm.sct_events.group_common_events import ignore_mutation_write_errors
+
+
+class MvSynchronousLongevityTest(LongevityTest):
+
+    def run_prepare_write_cmd(self):
+        with ignore_mutation_write_errors():
+            super().run_prepare_write_cmd()
+
+        self.log.debug("Start create_main_mv")
+        self.create_main_mv()
+        self.log.debug("Finish create_main_mv")
+
+    def test_mv_synch_updates_longevity(self):
+        with DbEventsFilter(db_event=DatabaseLogEvent.WARNING, line="perf_event_open"):
+            self.test_custom_time()
+
+    def create_main_mv(self):
+        """
+        Create materialized view for keyspace1.standard1 base table
+        """
+        with self.db_cluster.cql_connection_patient(self.db_cluster.nodes[0]) as session:
+            min_token, max_token = keyspace_min_max_tokens(node=self.db_cluster.nodes[0], keyspace="keyspace1")
+            self.log.debug("Partition ranges. Min: %s, max: %s", min_token, max_token)
+            if min_token is not None and max_token is not None:
+                token_range = max_token - min_token
+                # Create MV for part of the base table data
+                view_token_min = int(min_token + (token_range / 4))
+                view_token_max = int(max_token - (token_range / 4))
+                where = f"WHERE token(key) > {view_token_min} and token(key) < {view_token_max}"
+            else:
+                where = "WHERE key is not NULL"
+
+            main_mv_query = (f"CREATE MATERIALIZED VIEW keyspace1.main_mv_standard1 AS "
+                             f"SELECT * FROM keyspace1.standard1 "
+                             f"{where} PRIMARY KEY (key) WITH synchronous_updates = true")
+            self.log.debug("Start mv_standard1 materialized view creation with query: %s", main_mv_query)
+            session.execute(main_mv_query)
```

Comment:
I would suggest creating the table, with cl.ALL (we might issues related to how fast the schema sync)

Reply:
done